hieradata_aws/staging.yaml

```diff
@@ -90,7 +90,6 @@ govuk_mysql::server::slow_query_log: true
 govuk::node::s_asset_master::copy_attachments_hour: 7
 
 govuk::node::s_cache::real_ip_header: 'True-Client-Ip'
-govuk::node::s_cache::router_as_container: true
```

Comment:
Instead of reverting everything, could we just remove this line? It might be useful in the future.